README.md

```diff
@@ -74,16 +74,22 @@ Need:
 
 - Cake 2: <http://bakery.cakephp.org/articles/lorenzo/2011/10/16/cakephp_2_0_released>
 
+- Typo3 Flow 1: <http://en.wikipedia.org/wiki/TYPO3_Flow>
```

Comment:
Okay, maybe you'll not understand but this blog explains the release of Typo3-Flow (previously Flow or generally Flow3) on October 2011, also comparing it with Symfony...

http://t3n.de/magazin/flow3-vs-symfony2-kampf-giganten-229572/

Also, in the release history of Flow... http://typo3.org/about/the-history-of-typo3/ see the section in 2011.